src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsstafflookupservice/service/SearchStaffService.kt

```diff
@@ -9,7 +9,7 @@ import uk.gov.justice.digital.hmpps.hmppsstafflookupservice.db.repositories.Staf
 @Service
 class SearchStaffService(private val staffRepository: StaffRepository) {
   suspend fun searchStaff(email: String): Flow<StaffDetails> {
-    return staffRepository.findFirst20ByEmailLikeIgnoreCaseOrderByEmailAsc("%$email%@%")
+    return staffRepository.findFirst20ByEmailLikeIgnoreCaseOrderByEmailAsc("%$email%")
       .map { staffEntity -> StaffDetails(staffEntity.firstName, staffEntity.lastName, staffEntity.email, staffEntity.jobTitle) }
```

Comment:
I think this will definitely fix it.. although I think the JPA method name could be tweaked and get rid of the % ... https://docs.spring.io/spring-data/jpa/reference/jpa/query-methods.html - see the 'containing' example